### ISSUE TITLE
Item size minimum is 15min

### ISSUE THREAD

**Issue by Etienne-Larrouy** [namespace-ee/react-calendar-timeline#292]
Hello,

I wanted to display an item shorter than 15min but i'm figuring out it's not possible. I thought it was my custom calendar that was doing the problem. But i have the same behaviour with the very minimum example. I just changed the first item duration to 5min: 

```
const items = [
  {id: 1, group: 1, title: 'item 1', start_time: moment(), end_time: moment().add(5, 'minutes')},

  {id: 2, group: 2, title: 'item 2', start_time: moment().add(-0.5, 'hour'), end_time: moment().add(0.5, 'hour')},

  {id: 3, group: 1, title: 'item 3', start_time: moment().add(2, 'hour'), end_time: moment().add(3, 'hour')}
]
```

So, I want to display a 5 minutes item but it's 15min long.

![image](https://user-images.githubusercontent.com/9501482/39420155-8d02c1d8-4c63-11e8-9abc-e668d4332547.png)

The problem is not moment because in my custom app I'm using timestamp and i have the same problem.

I'm working with Ubuntu 64 bits and Mozilla Firefox

Thank you for your help :)

**Comment by Etienne-Larrouy** [comment]
Ok I found a fix. We need to set dragSnap to the minimum value we want. But it is a good fix ? 
Is the value of dragSnap intended to do this?

**Comment by mcMickJuice** [comment]
This should be fixed in 0.17.2
